sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/UsingCookies_NetCore.vb

```diff
@@ -8,7 +8,7 @@ Namespace Tests.Diagnostics
             ' Response headers
             response.Headers.Add("Set-Cookie", "") ' Noncompliant
             response.Headers("Set-Cookie") = "" ' Noncompliant
-            value = response.Headers("Set-Cookie") ' Noncompliant
+            value = response.Headers("Set-Cookie") ' Compliant
 
             ' Not the Set-Cookie header
             response.Headers.Add("something", "")
```

Comment:
as Vb.net is case insensitive, it would be nice to have tests for lower case method calls as well like

```
Dim responseHeaders2 = response.HEADERS
responseHeaders2.ADD("Set-Cookie", "") ' Noncompliant

Dim responseCookies2 = response.cookies
responseCookies2.append("", "") ' Noncompliant
```